snips_nlu/intent_parser/lookup_intent_parser.py

```diff
@@ -424,3 +425,9 @@ def _get_entity_placeholders(dataset, language):
 
 def _get_entity_name_placeholder(entity_label, language):
     return "%%%s%%" % "".join(tokenize_light(entity_label, language)).upper()
+
+
+def _convert_dict_keys_to_int(dct):
+    if isinstance(dct, dict):
+        return {int(k): v for (k, v) in dct.items()}
+    return dct
```

Comment:
Prefer the use of `iteritems(dct)` instead of `dct.items()` as the latter creates a copy of the (key, value) pairs while the former returns an iterator.
Side note: you don't need the parenthesis `(k, v)` => `k, v`